_api-files/embedding-api/objects/destinations/list-destinations.md

```diff
@@ -0,0 +1,52 @@
+---
+content-type: "embed-endpoint"
+endpoint: "destinations"
+key: "list-destinations"
+version: "3"
+order: 3
+
+
+title: "List destinations"
+method: "get"
+short-url: |
+  /v{{ object.version }}{{ object.endpoint-url }}
+full-url: |
+  {{ page.api-base-url }}{{ endpoint.short-url | flatify }}
+description: |
+  List the destination currently in use. Only a single data warehouse is supported per Stitch client account.
+
+
+returns: "An array (of length zero or one) of destinations."
+
+
+examples:
+  - type: "request"
+    language: "curl"
+    code: |
+      curl -H "Authorization: Bearer <ACCESS_TOKEN>" 
+           -H "Content-Type: application/json" 
+           -X {{ endpoint.method | upcase }} {{ endpoint.full-url | flatify | strip_newlines }}
+  - type: "response"
```

Comment:
Response is described as an array but this is an object.

Reply:
Fixed.